extension/jmxmetricextension/config.go

```diff
@@ -12,31 +12,45 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package jmxmetricsextension
+package jmxmetricextension
 
 import (
 	"fmt"
+	"strings"
 	"time"
 
 	"go.opentelemetry.io/collector/config/configmodels"
 )
 
 type config struct {
 	configmodels.ExtensionSettings `mapstructure:",squash"`
+	// The path for the JMX Metric Gatherer uber jar (/opt/opentelemetry-java-contrib-jmx-metrics.jar by default).
+	JarPath string `mapstructure:"jar_path"`
 	// The target JMX service url.
 	ServiceURL string `mapstructure:"service_url"`
-	// The script for the metric gatherer to run on the configured interval.
+	// The target system for the metric gatherer whose built in groovy script to run.  Cannot be set with GroovyScript.
+	TargetSystem string `mapstructure:"target_system"`
+	// The script for the metric gatherer to run on the configured interval.  Cannot be set with TargetSystem.
 	GroovyScript string `mapstructure:"groovy_script"`
-	// The duration in between groovy script invocations and metric exports.  Will be converted to milliseconds.
+	// The duration in between groovy script invocations and metric exports (10 seconds by default).
+	// Will be converted to milliseconds.
 	Interval time.Duration `mapstructure:"interval"`
 	// The JMX username
 	Username string `mapstructure:"username"`
 	// The JMX password
 	Password string `mapstructure:"password"`
-	// The otlp exporter timeout.  Will be converted to milliseconds.
+	// The metric exporter to use ("otlp" or "prometheus", "otlp" by default).
+	Exporter string `mapstructure:"exporter"`
+	// The Otlp Receiver endpoint to send metrics to ("localhost:55680" by default).
+	OtlpEndpoint string `mapstructure:"otlp_endpoint"`
```

Comment:
In theory, it should be `OTLPEndpoint`, but there are two other properties that are using `Otlp` instead.